Makefile

```diff
@@ -334,11 +334,20 @@ e2e-remove-resources:
 lint:
 	@echo "--> Running linter"
 	@go run github.com/golangci/golangci-lint/cmd/golangci-lint run --timeout=10m
+	@docker run -v $(PWD):/workdir ghcr.io/igorshubovych/markdownlint-cli:latest "**/*.md"
```

Comment:
This reads the .markdownlint file + .markdownlintignore file?

Reply:
Yep, it does!

We already use the same markdown linter in CI as a subcomponent of `super-linter`, I just pulled out specifically the markdown lint component and found a way to run it directly since:
1. that was what the issue was for
2. super linter is harder to run locally since it has so many moving pieces
3. running it would take really long for stuff that is already covered by our normal linter

Reply:
Sweet, lets file an issue to just make the linter run `make mdlint` as well then!